src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -223,6 +224,36 @@ private void CheckFileSystem()
             }
         }
 
+        private bool IsAppUsingManagedIdentity()
+        {
+            // If AzureWebJobsStorage__accountName is set, we are using identities.
+            if (!string.IsNullOrEmpty(_environment.GetEnvironmentVariable(AzureWebJobsStorageAccountName)))
+            {
+                return true;
```

Comment:
just curious - does this check work across SKUs? Please verify on windows dedicated SKU / EP SKU via private site extension - https://github.com/Azure/azure-functions-host/wiki/Deploying-the-Functions-runtime-as-a-private-site-extension

Reply:
Yes I used these steps - [link1](https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference?tabs=blob#connecting-to-host-storage-with-an-identity), [link2](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/use-managed-identity-instead-of-azurewebjobsstorage-to-connect-a/ba-p/3657606) with a windows dedicated .Net App to configure identity and the logic to check appsetting `AzureWebJobsStorage__accountName` for Managed identity is working. Let me know if you have concerns on this or there is another recommended way to detect MI case reliably.

There are other techniques mentioned [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference?tabs=blob#common-properties-for-identity-based-connections) for different cases like userAssigned, SystemAssigned. Should we include all of them in this check?

Still discussing with Bala offline to decide if we need this MI check here or remove it completely. Then I will circle back on how to reliably detect this case.